src/player/index.ts

```diff
@@ -233,7 +234,12 @@ function getNextTour(): CodeTour | undefined {
 
 function getHeaderText(step: CodeTourStep): string {
   if (isAccessibilitySupportOn() && step.line && step.file) {
```

Comment:
The check on line 238 seems redundant with line 236? it looks like you might've meant to also update the if statement in what's now line 236?

Reply:
resolving :)